### PR TITLE
[Dup of #2785] [CI][test-perf] MIOpenDriver to use rocrand to init buffers. Do not init output buffers. Use non-DEV build in perf test. 

### PR DESCRIPTION
**Pull request by junliume**
@atamazov  #2785 

### NOT TO BE MERGED ###

used to kick off CI

### CODE REVIEW

**Comment by junliume** [comment]
@atamazov :
```
[2024-03-05T18:51:48.341Z] org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:

[2024-03-05T18:51:48.341Z] WorkflowScript: 61: expecting EOF, found 'else' @ line 61, column 7.

[2024-03-05T18:51:48.341Z]        } else {

[2024-03-05T18:51:48.341Z]          ^
```

File: Jenkinsfile [hunk 1]
```diff
@@ -56,7 +56,7 @@ def cmake_build(Map conf=[:]){
     {
         make_targets = 'install ' + make_targets
         setup_args = " -DBUILD_DEV=Off -DCMAKE_INSTALL_PREFIX=${miopen_install_path}" + setup_args
-    } else if(package_build == true)
+    } else if(package_build == true) {
```
Comment (atamazov) [inline Jenkinsfile:59]: @junliume Thanks, I'll update the original branch.

**Comment by atamazov** [comment]
@junliume @JehandadKhan I think it is better to review #2785 as I am unable to update the branch in this repo (and thus I can't resolve review comments in this PR)

File: CMakeLists.txt [hunk 2]
```diff
@@ -485,10 +489,19 @@ if(MIOPEN_USE_ROCBLAS)
     set(MIOPEN_PACKAGE_REQS "${MIOPEN_PACKAGE_REQS}, rocblas")
 endif()
 
+# PR #2785 MIOpenDriver to use rocrand to init buffers
+set(MIOPEN_PACKAGE_REQS "${MIOPEN_PACKAGE_REQS}, rocrand")
+
```
Comment (atamazov) [inline CMakeLists.txt]: @junliume This is duplicate of lines 499-504, but not protected with if(MIOPEN_BUILD_DRIVER).

File: Dockerfile [hunk 3]
```diff
@@ -50,6 +50,7 @@ DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-unauthenticated \
     python3-pip \
     python3-venv \
     rocblas \
+    rocrand \
```
Comment (atamazov) [inline Dockerfile]: @junliume In the `ci_d868bf` docker, rocrand is included with ROCm 6.0 installation, so this line is not necessary.

File: CMakeLists.txt (hunk 2 above)
Comment (junliume) [reply CMakeLists.txt]: @atamazov yes this line should be removed, I forgot to update the branch.

File: Dockerfile (hunk 3 above)
Comment (junliume) [reply Dockerfile]: @atamazov I'm investigating it. Meanwhile I try to update CI docker to ROCm 6.1 RC2, and a few things are missing there.

Comment (junliume) [reply Dockerfile]: Reverted these changes, restarted CI and let's see, I recall an error reporting rocrand is missing.

**Pull request closed without merging**